inspire/modules/formatter/views.py

```diff
@@ -84,3 +88,36 @@ def get_latex_file(recid, latex_format):
             )
         }
     )
+
+
+@blueprint.route("/export-as/<path:export_format>/<path:ids>",
```

Comment:
So I think up to 5000 ids could be served (as there are e.g. certain papers having ~3000 references). After that we can truncate the export and add an appropriate comment line (e.g. in BibTeX for the BibTeX format) describing the truncation being implemented and to contact us if there are further needs.

Regarding permission, luckily there are no real secret records so far in INSPIRE. Rather some hidden fields and in case some protected fulltext files, but these are unrelated with this very specific PR.